### PR TITLE
Add in Hyperlint Base Style Guide + Spellcheck Configurations

### PR DESCRIPTION
**Pull request by bllchmbrs**
## Change Summary

This adds in the base Hyperlint configurations and shows where to add files to effectively configure your style guide and rules.

## Related issue number

NA

## Checklist

* [x] The pull request title is a good summary of the changes - it will be used in the changelog
* [-] Unit tests for the changes exist
* [-] Tests pass on CI
* [-] Documentation reflects the changes where applicable
* [x] My PR is ready to review, **please add a comment including the phrase "please review" to assign reviewers**

Please review @sydney-runkle

### CODE REVIEW

File: .hyperlint/style_guide_test.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (bllchmbrs) [inline .hyperlint/style_guide_test.md:1]: This file will run the checks (but no recommendations) use it to iterate on the styles / dictionary. It doesn't need to be committed to the repo (but can be).

File: .hyperlint/.vale.ini [hunk 2]
```diff
@@ -0,0 +1,7 @@
+StylesPath = styles
```
Comment (bllchmbrs) [inline .hyperlint/.vale.ini:1]: This is the base style guide check cconfiguration, no action needed here on your part - this won't change much or often.

File: .hyperlint/styles/config/vocabularies/hyperlint/accept.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+validator
```
Comment (bllchmbrs) [inline .hyperlint/styles/config/vocabularies/hyperlint/accept.txt:1]: This is your custom dictionary: [Read more here.](https://docs.hyperlint.com/ai-reviewer/custom-style-guide)

File: .hyperlint/styles/hyperlint/repeatedWords.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+extends: repetition
```
Comment (bllchmbrs) [inline .hyperlint/styles/hyperlint/repeatedWords.yml:1]: This is a custom rule, lots of possibilities here but best to start simple. [Read more here](https://docs.hyperlint.com/ai-reviewer/custom-style-guide)

**Comment by bllchmbrs** [comment]
> ## style_guide_test.md review details
> ### Line Edit Problems/Proposals
> _Note: The following 3 comments are grouped by line._
> 
> ### Line 3
> **Existing text:**
> 
> # This ia a test file
> **Issues:**
> 
> * Style Guide - (Spelling-error) Did you really mean 'ia'?
> 
> ### Line 5
> **Existing text:**
> 
> it will flag errors like on pydantic.
> **Issues:**
> 
> * Style Guide - (Terms-error) Use 'Pydantic' instead of 'pydantic'.
> 
> ### Line 11
> **Existing text:**
> 
> This is is an issue.
> **Issues:**
> 
> * Style Guide - (Repetition-error) 'is' is repeated!
> * Style Guide - (hyperlint.repeatedWords-error) 'is' is repeated, did you mean to repeat this word?

As you make changes, you'll get more comments like this. If the style guide test file is present on a pull request, then it will post a comment like that on every synchronization.

File: .hyperlint/styles/config/vocabularies/hyperlint/accept.txt (hunk 3 above)
Comment (sydney-runkle) [reply .hyperlint/styles/config/vocabularies/hyperlint/accept.txt:1]: Great! I'll expand on this when I do some docs updates tomorrow.

**Comment by sydney-runkle** [comment]
cc @Viicos, this could be a great way to enforce Pydantic and `pydantic` across the docs :)

**Pull request merged**